src/tests/unit/tests/DetailsView/components/save-assessment-dialog.test.tsx

```diff
@@ -60,7 +63,19 @@ describe('SaveAssessmentDialog', () => {
         render(<SaveAssessmentDialog {...propsStub} />);
         expect(getMockComponentClassPropsForCall(Dialog).hidden).toEqual(true);
     });
+});
+
+describe('dialog interaction', () => {
+    beforeEach(() => {
```

Comment:
the flow of the interaction tests is confusing me. I'd expect it to:
1. create a variable for `wrapper` inside the `describe` block but before the `beforeEach`,
2.  call `useOriginalReactElements`, set `propsStub.isOpen`, and set `wrapper` (to the result of the `render` call) in the `beforeEach` 
3. and then do the actual interaction and assertions in the tests

right now this is still duplicating a lot of code that doesn't need to be

Reply:
hi @madalynrose fixed it please re-review